### ISSUE TITLE
How does TensorFlow calculate the gradients of an FFT layer?

### ISSUE THREAD

**Issue by zhao62** [tensorflow/tensorflow#34785]
Hi, if I insert the function, i.e., tf.fft(input, name=None), into a neural network, how does TensorFlow calculate the gradients in backpropagation?

I didn't find a documentation about this. I am using TensorFlow 1.0.

Does anyone know？

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
 Thanks!
